packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,352 @@
+  <Item textValue="Row with link, icon, and description">
+    <Folder />
+    <Link>Row with link, icon, and description</Link>
+    <Text slot="description">Description of row</Text>
+  </Item>
+  <Item textValue="Row with action button">
+    Row with action button
+    <ActionButton aria-label="Info">
+      <Info />
+    </ActionButton>
+  </Item>
+  <Item textValue="Row with action group">
+    Row with action group
+    <ActionGroup buttonLabelBehavior="hide">
+      <Item key="edit">
+        <Edit />
+        <Text>Edit</Text>
+      </Item>
+      <Item key="delete">
+        <Delete />
+        <Text>Delete</Text>
+      </Item>
+    </ActionGroup>
+  </Item>
+  <Item textValue="Row with action menu">
+    Row with action menu
+    <ActionMenu>
+      <Item key="edit" textValue="Edit">
+        <Edit />
+        <Text>Edit</Text>
+      </Item>
+      <Item key="delete" textValue="Delete">
+        <Delete />
+        <Text>Delete</Text>
+      </Item>
+    </ActionMenu>
+  </Item>
+</ListView>
+```
+
+## Selection
+
+By default, ListView doesn't allow row selection but this can be enabled using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected rows. Note that the value of the selected keys must match the `key` prop of the Item.
+
+The example below enables multiple selection mode, and uses `defaultSelectedKeys` to select the rows with keys "row1" and "row3".
+
+```tsx example
+<ListView maxWidth="size-6000" selectionMode="multiple" defaultSelectedKeys={["Charizard", "Venusaur"]} aria-label="List selection example">
+  <Item key="Charizard" textValue="Charizard">
+    <Content>Charizard</Content>
+  </Item>
+  <Item key="Blastoise" textValue="Blastoise">
+    <Content>Blastoise</Content>
+  </Item>
+  <Item key="Venusaur" textValue="Venusaur">
+    <Content>Venusaur</Content>
+  </Item>
+  <Item key="Pikachu" textValue="Pikachu">
+    <Content>Pikachu</Content>
+  </Item>
+</ListView>
+```
+
+### Controlled selection
+
+To programmatically control row selection, use the `selectedKeys` prop paired with the `onSelectionChange` callback. The `key` prop from the selected rows will
+be passed into the callback when the row is pressed, allowing you to update state accordingly.
+
+Here is how you would control selection for the above example.
+
+```tsx example export=true
+function PokemonList(props) {
+  let rows = [
+    {id: 1, name: 'Charizard'},
+    {id: 2, name: 'Blastoise'},
+    {id: 3, name: 'Venusaur'},
+    {id: 4, name: 'Pikachu'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <ListView items={rows} maxWidth="size-6000" aria-label="List with controlled selection" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} {...props}>
+      {(item) => (
+        <Item textValue={item.name}>
+          {item.name}
+        </Item>
+      )}
+    </ListView>
+  );
+}
+```
```

Comment:
I continue to be surprised that you can declare a function component in a `tsx example` and it renders without explicitly calling it.

Reply:
Oh, looking at ActionBar docs we should explicitly call the component.
```suggestion

<PokemonList />
```
```